client/devdocs/design/index.jsx

```diff
@@ -141,6 +142,7 @@ let DesignAssets = React.createClass( {
 					<SelectDropdown searchKeywords="menu" />
 					<SocialLogos />
 					<Spinner searchKeywords="loading" />
+					<SpinnerButton />
```

Comment:
As this is related to other spinners which specify a `searchKeywords`, I might also expect the same `searchKeywords="loading"`